tachyon/crypto/commitments/vector_commitment_scheme.h

```diff
@@ -22,6 +25,17 @@ class VectorCommitmentScheme {
     return base::bits::SafeLog2Ceiling(derived->N());
   }
 
+  BatchCommitmentState& batch_commitment_state() {
+    return batch_commitment_state_;
+  }
+  bool batch_mode() const { return batch_commitment_state_.batch_mode; }
+
+  void set_batch_mode(size_t batch_count) {
```

Comment:
Since this is not an obvious setter, please change it to `SetBatchMode(size_t batch_count)`

```suggestion

```